cmd/pulumi-test-language/tests.go

```diff
@@ -28,19 +28,24 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-type languageTest struct {
+type testRun struct {
+	subDir string
```

Comment:
I think I would have made this implicit. It's always painful trying to keep data in sync between two places and named subdirs also don't let you see at a glance what order the test is going to run in. 

I'd remove subDir and just name the folders "0","1", etc. With a special case for when len(runs) == 1 that there is no integer named directory, the test is just at the top level.

Reply:
Good idea :heavy_check_mark: